lib/modules/manager/flux/common.ts

```diff
@@ -8,3 +12,87 @@ export const systemManifestHeaderRegex =
 export function isSystemManifest(file: string): boolean {
   return regEx(systemManifestFileNameRegex).test(file);
 }
+
+export function resolveHelmReleaseManifest(
+  resource: HelmRelease,
+  helmRepositories: HelmRepository[],
+  helmCharts: HelmChart[],
+): {
+  name: string;
+  dep: PackageDependency;
+  matchingRepositories: HelmRepository[];
+} | null {
+  if (resource.spec.chartRef) {
```

Comment:
invert condition to reduce nesting.

Reply:
not sure what you mean here? I can do a `if (!resource.spec.chartRef) { return null; }`, but note that there's an `else if (resource.spec.chartRef) {` later, so I don't think the condition here can be dropped completely. so the nesting level would remain unchanged anyway? unless I'm missing something on how you suggest to update the condition?